nestkernel/universal_data_logger.h

```diff
@@ -290,6 +302,273 @@ nest::UniversalDataLogger< HostNode >::DataLogger_::DataLogger_(
   recording_interval_ = req.get_recording_interval();
   recording_offset_ = req.get_recording_offset();
 }
+
+
+/**
+ * Universal data-logging plug-in for neuron models.
+ *
+ * This class provides logging of universal data such as
```

Comment:
You should explain early on the difference between the normal and the dynamic logger.

Reply:
Done in 81e1178